docs/01-overview/main-areas/eventing/evnt-02-useful-links.md

```diff
@@ -6,9 +6,9 @@ Find out more about Eventing by following the links on this page.
 
 To learn more about how Eventing works, see:
 
-- [Eventing architecture](../../../05-technical-reference/03-architecture/evnt-01-architecture.md) - describes how Eventing works and the main actors involved, such as the Eventing Controller and Event Publisher Proxy. You can also find more information about event types here.
+- [Eventing architecture](../../../05-technical-reference/00-architecture/evnt-01-architecture.md) - describes how Eventing works and the main actors involved, such as the Eventing Controller and Event Publisher Proxy. You can also find more information about event types here.
 - [Event processing and delivery](../../../05-technical-reference/evnt-01-event-processing.md) - contains a diagram and explanation of the the Eventing processing and delivery flow.
-- [Subscription CRD](../../../05-technical-reference/06-custom-resources/evnt-01-subscription.md) - describes the `Subscription` CRD which you need to subscribe to events.
+- [Subscription CR](../../../05-technical-reference/00-custom-resources/evnt-01-subscription.md) - describes the `Subscription` CRD which you need to subscribe to events.
```

Comment:
```suggestion
- [Subscription CR](../../../05-technical-reference/00-custom-resources/evnt-01-subscription.md) - describes the `Subscription` custom resource which you need to subscribe to events.
```
It should be CR.
And `custom resource` instead of CR, because otherwise you'd have `Subcription CR - describes Subscription CR`.

Reply:
😆